### PR TITLE
Draw LineChart hover attributes

### PR DESCRIPTION
**Pull request by KaroMourad**
Add base mouseover functionality:
  - draw nearest points on hover
  - set nearest points style
  - find closest point on hover
  - set closest point style
  
 Add packages: 
  - bs58check (used by "JS standard code style", popular)
 
 Add hashing functions:
  - contextToHash
  - traceToHash

### CODE REVIEW

File: aim/web/ui_v2/src/components/LineChart/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.HoverLine {
```
Comment (roubkar) [inline aim/web/ui_v2/src/components/LineChart/styles.module.css:1]: @KaroMourad can you please change this file name.
I think `LineChart.module.css` would be more helpful

**Review by roubkar — CHANGES_REQUESTED** [review]
Please change the styles module file name, and resolve conflicts with `main` branch

**Review by roubkar — APPROVED** [review]
lgtm

File: aim/web/ui_v2/src/utils/toHash.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+export function bufferToString(buffer: Buffer): string {
+  return buffer.toString('utf-8');
+}
+
+export function contextToHash(context: { [key: string]: unknown }): string {
+  return encode(Buffer.from(JSON.stringify(context)));
+}
+
+export function traceToHash({
```
Comment (gorarakelyan) [inline aim/web/ui_v2/src/utils/toHash.ts:11]: Probably it is better to have one encoder which receives any object(s) and returns encoded string.

**Review by gorarakelyan — APPROVED** [review]

File: aim/web/ui_v2/src/utils/toHash.ts (hunk 2 above)
Comment (roubkar) [reply aim/web/ui_v2/src/utils/toHash.ts:11]: yeah, definitely! @KaroMourad can you please add a separate utility function for encoding and use it in hashing functions?

Comment (KaroMourad) [reply aim/web/ui_v2/src/utils/toHash.ts:11]: Fine , i will fix this

**Pull request merged**